src/modules/person/api/personenkontext.controller.ts

```diff
@@ -62,4 +69,41 @@ export class PersonenkontextController {
             throw error;
         }
     }
+
+    @Get()
+    @ApiOkResponse({
+        description: 'The personenkontexte were successfully returned.',
+        type: [PersonendatensatzResponse],
+        headers: PagingHeadersObject,
+    })
+    @ApiBadRequestResponse({ description: 'Request has wrong format.' })
+    @ApiUnauthorizedResponse({ description: 'Request is not authorized.' })
+    @ApiNotFoundResponse({ description: 'The personenkontexte were not found.' })
+    @ApiForbiddenResponse({ description: 'Insufficient permissions to perform operation.' })
+    @ApiInternalServerErrorResponse({ description: 'An internal server error occurred.' })
+    public async findPersonenkontexte(
+        @Query() queryParams: PersonenkontextQueryParams,
+    ): Promise<PagedResponse<PersonenkontextdatensatzResponse>> {
+        const findPersonenkontextDto: FindPersonenkontextDto = this.mapper.map(
+            queryParams,
+            PersonenkontextQueryParams,
+            FindPersonenkontextDto,
+        );
+
+        const result: Paged<PersonenkontextDto> = await this.personenkontextUc.findAll(findPersonenkontextDto);
+
+        const responseItems: PersonenkontextdatensatzResponse[] = this.mapper.mapArray(
+            result.items,
+            PersonenkontextDto,
+            PersonenkontextdatensatzResponse,
+        );
+        const response: PagedResponse<PersonenkontextdatensatzResponse> = new PagedResponse({
+            items: responseItems,
+            total: result.total,
+            offset: result.offset,
+            limit: result.limit,
+        });
```

Comment:
Between the consts we don't need blank lines.

Reply:
Done